lib/geocoder/lookups/location_iq.rb

```diff
@@ -7,23 +7,35 @@ def name
       "LocationIq"
     end
 
+    def use_ssl?
+      true
```

Comment:
The documentation examples on documentation link seems to only show http examples. Are you sure that https works?

Reply:
Yes, `https` works along with `http`

Reply:
If it works with both, should we let the user decide which one to use? It's available as a config setting in geocoder

Reply:
@TrangPham @alexreisner Shall I update this to `["use_ssl"]` and let the users decide?

Reply:
@SrihariThalla definitely. Thanks!